v2/spiffetls/tlsconfig/config_test.go

```diff
@@ -0,0 +1,687 @@
+package tlsconfig_test
+
+import (
+	"bufio"
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"github.com/spiffe/go-spiffe/v2/bundle/x509bundle"
+	"github.com/spiffe/go-spiffe/v2/internal/test"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/spiffetls/tlsconfig"
+	"github.com/spiffe/go-spiffe/v2/svid/x509svid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestTLSClientConfig(t *testing.T) {
+	// Create trust domain and bundle
```

Comment:
nit: the comments in these config tests are pretty redundant with the code itself and aren't really adding any clarity to the code. The newlines are nice to break up groups of functionality, but I don't know that we need all of the comments.

Reply:
done